app/code/Magento/Sales/Api/Data/TrackInterface.php

```diff
@@ -31,6 +31,21 @@ public function setTrackNumber($trackNumber);
     public function getTrackNumber();
 
     /**
+     * Sets the track URL for the shipment package.
+     *
+     * @param string $trackUrl
+     * @return $this
+     */
+    public function setTrackUrl($trackUrl);
```

Comment:
New methods cannot be added into existing `@api` interfaces as it will break every existing interface implementation. Please check http://devdocs.magento.com/guides/v2.0/contributor-guide/backward-compatible-development/#introduction-of-a-method-to-a-class-or-interface